libexec/ipmi

```diff
@@ -78,10 +78,10 @@ usage()
 {
   echo "(c) 2008-2011 Alces Software Ltd & Stephen F Norledge"
   echo "alces-ipmi command, perform ipmi commands on single or multiple machines"
-  echo "usage: alces ipmi [-g] <name> -c <command>"
```

Comment:
Their are still a few edge cases in which the bash version of the usage will be displayed. The only way I could see to get around it is to rewrite it in ruby